src/entity/task/vdaf.rs

```diff
@@ -0,0 +1,67 @@
+use crate::json_newtype;
+use serde::{Deserialize, Serialize};
+use std::collections::HashSet;
+use validator::{Validate, ValidationError, ValidationErrors};
+
+#[derive(Serialize, Deserialize, Validate, Debug, Clone, Eq, PartialEq)]
+pub struct Histogram {
+    #[validate(required, custom = "sorted", custom = "unique")]
+    pub buckets: Option<Vec<i32>>,
+}
+
+fn sorted(buckets: &Vec<i32>) -> Result<(), ValidationError> {
```

Comment:
nit: this method makes a clone (allocating space), and sorts (taking `O(n lg n)` time).

what we're trying to do is determine if the list is already sorted; instead, consider just iterating through the elements and checking for that they are ordered (pairwise), which doesn't require an allocation and runs in linear time.

Reply:
Are there going to be enough buckets we care about the computation in comparison to the network and db io?

Reply:
Opened https://github.com/divviup/divviup-api/issues/102 as this is not really related to this PR aside from having moved these lines between files